app/assets/src/components/views/report/TaxonModal.jsx

```diff
@@ -140,82 +136,82 @@ class TaxonModal extends React.Component {
   }
 
   render() {
+    // The Modal component always expects a 'trigger', even though it doesn't make sense in our use case.
+    // <TaxonModal> is always open when rendered. To hide <TaxonModal>, we simply don't render it.
     return (
       <Modal
         title={this.props.taxonName}
-        trigger={<span onClick={this.handleOpen}>{this.props.trigger}</span>}
-        open={this.state.open}
-        onClose={this.handleClose}
+        trigger={<span />}
+        open
+        onClose={this.props.handleClose}
```

Comment:
Should we add a default behavior that closes the modal? Or is this incompatible with your solution?